pkg/foghorn/controller.go

```diff
@@ -384,6 +384,13 @@ func (c *Controller) reportStatus(ns string, activity *jxv1.PipelineActivity, jo
 	}
 	urlBase := c.getReportURLBase()
 	if urlBase != "" {
+		urlTeam := c.getReportURLTeam()
+		team := ns
+		// override with env var if set
```

Comment:
Why do we override with env var if it set?

Reply:
Because the namespace is the proper default.